### ISSUE TITLE
TorrentRSS (Generic)

### ISSUE THREAD

**Issue by Minxster** [Prowlarr/Prowlarr#1737]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Where is the TorrentRSS feature within Prowlarr? According to the wiki there should be an ability to add TorrentRSS feeds?

[https://wiki.servarr.com/prowlarr/faq#can-i-add-any-generic-torrent-rss-feed][(url)](https://wiki.servarr.com/prowlarr/faq#can-i-add-any-generic-torrent-rss-feed)

Is this a bug that stops the option from appearing? Or is this a feature that does no exit/removed/still to come?

### Expected Behavior

Have the ability to add RSS feeds into Prowlarr

### Steps To Reproduce

Attempt to add a generic torrent RRS or TorrentRSS indexer into Prowlarr

### Environment

```markdown
- OS:Ubuntu 20.04
- Prowlarr: 1.5.2.3484
- Docker Install: Yes, Portainer
- Using Reverse Proxy: Nginx Proxy Manager
- Browser: Chrome
```

### What branch are you running?

Master

### Trace Logs?

N/A

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [X] I have followed the steps in the wiki link above and provided the required trace logs that are relevant and show this issue.

**Comment by Minxster** [comment]
Thanks for the automated response, but this is a bug if it's supposed to be in the UI but it doesn't show up? I'm basically trying to use a feature that should exist, according to the wiki, but in actuality it doesn't seem to be available?

For reference, I've already asked on/in the Subreddit, and it's been over 14 days with no responce?

**Comment by Minxster** [comment]
@mynameisbogdan thanks for taking the time to reply to me about this 😉. Do we know if this works as I'm anticipating? In that it'll download the RSS feed and allow Prowlarr to cache these results and feed then out upon request? And/Or is it a separate RSS feed given to *arr apps as and when they request it?

I'll spin up another container and take a look at the develop + nightly builds 👍

**Comment by Minxster** [comment]
Ok, thanks, so not quite what I was hoping for, but at least it's something. Last question and I'll shut-up after this, do you know if Prowlarr validates SSL certificates? Could I build a HTTPS proxy and feed that to Prowlarr in general?

I know it seems overkill, I'm just trying to minimize my footprint and requests to 3rd parties. I have a huge back catalogue that all need work but I'll killing my connections with "too many requests". I know making duplicate requests (during my migration), so I'm trying to find geeky ways to reduce this?

**Comment by Minxster** [comment]
Disabling SSL certs is always a bad move 👍😉 I'll only be doing it for testing purposes while I get things figured out. After that I can look at hardening everything again.

I'm not too sure on TTL, to start with I'll probably have a ludicrous time of ~8hrs. The RSS feed I'm using is a full 24hr catalogue, so some arbitrary value between 12 to 24 would work for me. That said I also need to control general API (non-RSS) connections as I'm smashing those connections during my testing and migration. So also having that at a silly high number to begin with would help/work.

It's all because of the migration process, I have multiple systems all chatting, asking for the same stuff. I just need to calm it all down and cache results. I initially hoped that Prowlarr would do this out of the box, e.g. "something asked for this 30mins ago, my config is set to 60mins for a duplicate request, pass on previous result(s)", but it's a very specific edge-case, so understand why this would be the case 😁

**Comment by Minxster** [comment]
> Also Torrent RSS won't work with searches, only keyword-less searches aka RSS syncs. So this might not be what you're looking for. :)

Newly coming to *arr, I didn't realise the ethos of how it worked until I started testing. It's not how my other systems worked in the past. Initially I wasn't impressed, but then I realised it's advantages 😉. It is actually these RSS syncs I want to cache, for now, as I have too many systems/containers polling it/them. No one needs over 8 file management apps/solutions, but that's where I'm at with testing 🤣
